tests/e2e/README.md

```diff
@@ -53,10 +53,10 @@ You can run the e2e tests locally using this command:
 yarn test-e2e
 ```
 
-Puppeteer runs headless by default (i.e. browser is not visible). However, sometimes it's useful to observe the browser while running tests. To see the browser window and the running tests you can pass `PUPPETEER_HEADLESS=false` as follows:
+Puppeteer runs headless by default (i.e. browser is not visible). However, sometimes it's useful to observe the browser while running tests. To see the browser window and the running tests you can pass `HEADLESS=false` as follows:
```

Comment:
There are various references to Puppeteer still in the codebase and documentation.

Reply:
Indeed, I left a lot of those in documentation blocks mostly. I pushed a change which replaced all except one for a piece o code that still needs to be evaluated if it should stay or go.